spec/README.md

```diff
@@ -67,6 +67,7 @@ Example structure for the German language file, which only contains lower case c
         "alphabet": "Latn",
         "continent": "EU",
         "language": "German",
+        "variant": "",
```

Comment:
It'd be wrong to say that it's not necessary. Instead, it must no be specified if it's empty. Otherwise the schema validation will fail. So I see two ways:

- You're going to remove it entirely
- You're commenting it out